bitnami/prometheus/templates/server/clusterrole.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if .Values.rbac.create }}
+kind: ClusterRole
+apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
+metadata:
+  name: {{ include "common.names.fullname" . }}
```

Comment:
If this is inside the "server" section, shouldn't we use the naming of the server instead of the chart?
Also, if this is a cluster-wide resource, then you should use *.fullname.namespace to avoid issues with the parallel
execution of VIB. This also ensures that you can use the same release name in different namespaces